src/main/java/util/Maths.java

```diff
@@ -0,0 +1,53 @@
+package util;
+
+public class Maths {
+	public static double clamp(double x, double min, double max) {
+		if (x < min) return min;
+		if (x > max) return max;
```

Comment:
The second `if` could just be `return Math.min(max, x)`, probably personal preference which one looks better.

Reply:
I personally prefer the symmetry of the two if-statements next to each other, but yeah, we could use Math.min too